frontend/src/components/HandbookOverlay/HandbookOverlay.css

```diff
@@ -26,7 +26,8 @@
 	padding: 0 1.7rem;
```

Comment:
1.7rem? What happened to our equivalent pixel-sized rems? This should be 1.75rem.

Also, it's a bit confusing to use the shorthand `padding` property with two values, and then override padding-bottom. Better to use just `padding: 0 1.75rem 0.5rem`